activities/management/commands/jslps_mediator_new.py

```diff
@@ -149,27 +164,34 @@ def handle(self, *args, **options):
 			if animator != None:
 				jslps_animator_list = JSLPS_Animator.objects.filter(animator_code=ac)
 				if len(jslps_animator_list) == 0:
-					jslps_animator = JSLPS_Animator(animator_code=ac,animator=animator)
-					jslps_animator.save()
+					jslps_animator, created = \
+						JSLPS_Animator.objects.get_or_create(animator_code=ac,
+									   						 animator=animator,
+									   						 activity="LIVELIHOOD",
+									   						 user_created_id=user_obj.id)
 				else:
 					jslps_animator = jslps_animator_list[0]
 					jslps_animator.animator = animator
 					jslps_animator.save()
 			else:
 				animator_list = Animator.objects.filter(name = an,
-									gender = gender,
-									partner = partner,
-									district = district.district)
+														gender=gender,
+														partner=partner,
+														district=district.district)
 				if len(animator_list) != 0:
 					animator = animator_list[0]
 					jslps_animator_list = JSLPS_Animator.objects.filter(animator_code=ac,animator=animator)
 					if len(jslps_animator_list) == 0:
-						jslps_animator = JSLPS_Animator(animator_code=ac,animator=animator)
-						jslps_animator.save()
+						jslps_animator, created = \
+						JSLPS_Animator.objects.get_or_create(animator_code=ac,
+									   						 animator=animator,
+									   						 activity="LIVELIHOOD",
+									   						 user_created_id=user_obj.id)
 					else:
 						jslps_animator = jslps_animator_list[0]
 						if jslps_animator.animator == None:
 							jslps_animator.animator = animator
 							jslps_animator.save()
 				else:
 					wtr.writerow(['Animator not saved and duplicate also not exist',ac, "not saved"])
+
```

Comment:
Please close all csv files in all scripts

Reply:
ok